Framework/include/QualityControl/runnerUtils.h

```diff
@@ -81,6 +81,20 @@ inline int computeRunNumber(const framework::ServiceRegistry& services, const bo
   return run;
 }
 
+/**
+ * If the string is longer than the requested length, truncate it.
+ * @param toBeTruncated
+ * @return toBeTruncated truncated to `length` characters
+ */
+inline std::string truncate(std::string toBeTruncated, size_t length)
```

Comment:
Doesn't `string::substr()` do exactly that? It should survive length() > length

Reply:
I thought not but let me give it a try !

Reply:
Returns a substring [pos, pos+count). If the requested substring extends past the end of the string, or if count == npos, the returned substring is [pos, size()). 
from https://en.cppreference.com/w/cpp/string/basic_string/substr

Reply:
Dammit, you were right !
Fixed;